ethportal-api/src/dashboard/grafana.rs

```diff
@@ -5,7 +5,7 @@ use std::fs;
 use ureq;
 
 pub const DASHBOARD_TEMPLATES: &[&str] =
-    &["../etc/grafana/dashboards/collected-metrics-dashboard.json.template"];
+    &["./etc/grafana/dashboards/collected-metrics-dashboard.json.template"];
```

Comment:
Do we have to add the trin-bench metrics dashboard here? Seems like the template file was added, but is never created?

Reply:
The `trin-bench` template is really only meant as a backup for the `trin-bench.ethdevops.io` grafana process. Since it has a different configuration from the `collected-metrics-dashboard`, it's not useful to anybody running grafana locally, so I don't think there's any need to create it locally.